samples/model_repository/vllm_model/1/model.json

```diff
@@ -1,6 +1,7 @@
 {
     "model":"facebook/opt-125m",
-    "disable_log_requests": "true",
+    "disable_log_requests": true,
     "gpu_memory_utilization": 0.5,
-    "enforce_eager": "true"
+    "enforce_eager": true,
+    "disable_log_stats": false
```

Comment:
We should add `disable_log_stats` to sample `model.json`. Open to suggestions.

Reply:
you can add this during test initialization

Reply:
Discussed offline. Adding `disable_log_stats` to sample config to let user aware of the change/metrics.